pkgs/by-name/hi/hidviz/package.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, fetchFromGitHub
+, cmake
+, pkg-config
+, stdenv
+# Package dependencies
+, qt6
+, libusb1
+, protobuf
+, asio
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hidviz";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "hidviz";
+    repo = pname;
```

Comment:
(I also won't block merge over this, but using `repo = pname` is discouraged for newer packages as well, since it's a bit of a useless use of `rec` anyway, doesn't save any time or energy or space, and if, for example, a downstream derivation overrides this one but changes the pname (e.g. to add `-extras`), it will unexpectedly stop working)